src/planscape/plan/views.py

```diff
@@ -205,31 +205,23 @@ def _validate_constraint_values(max_budget, max_treatment_area_ratio, max_road_d
             "Max slope must be a number value >= 0.0")
 
 
-def _save_project_parameters(body, project: Project):
-    max_budget = body.get('max_budget', None)
-    max_treatment_area_ratio = body.get('max_treatment_area_ratio', None)
-    max_road_distance = body.get('max_road_distance', None)
-    max_slope = body.get('max_slope', None)
-
-    _validate_constraint_values(
-        max_budget, max_treatment_area_ratio, max_road_distance, max_slope)
-
+def _set_project_parameters(max_budget, max_treatment_area_ratio, max_road_distance,
```

Comment:
Are any tests for create_project and update_project affected?

Reply:
No changes in test behavior were intended.